workflows/theiacov/wf_theiacov_illumina_pe.wdl

```diff
@@ -155,6 +155,14 @@ workflow theiacov_illumina_pe {
             seq_method = seq_method
         }
       }
+      # Calculate the percentage of mapped reads for flu samples
```

Comment:
Can you move this block (158-165) into the wf_flu_track.wdl where we put all the flu-specific items? 

I like how you are checking if the bam files are defined.